mlrun/db/httpdb.py

```diff
@@ -471,11 +502,15 @@ def remote_status(self, kind, selector):
 
         return resp.json()['data']
 
-    def submit_job(self, runspec, schedule=None):
+    def submit_job(
```

Comment:
@Hedingber how do we handle errors and repeated errors in scheduled tasks?

Reply:
same